README.md

```diff
@@ -58,7 +58,7 @@ Adding the Septic extension to VS Code allows you to do the following when loadi
     -   `ctrl-k` `ctrl-2` folds all level 2 sections (e.g. `SopcMvr` and `CalcModl`) and lower except the level 2 section at the cursor.
     -   `ctrl-k` `ctrl-3` folds all level 3 sections (e.g. `CalcPvr`) and lower except the level 3 section at the cursor.
     -   More info on folding [here](https://code.visualstudio.com/docs/editor/codebasics#_folding)
--   Diagnostics (Errors/Warnings) for a certain line can be disabled by writing `// noqa` or `{# noqa #}` at the same line. It is recommended to use the jinja version when suppressing diagnostics when using scg in order to get proper diagnostics on the generated config. To disable certain errors/warnings write `{# noqa: Ennn, Wnnn #}` or `// noqa: Ennn, Ennn` (i.e. `noqa` + `:` + comma separated list of errors). The relevant error codes are displayed on the errors/warnings (e.g. `septic Enn`). To disable all diagnostics, use the `septic.diagnostics.enabled` setting. Diagnostics can be ignored for certain files by adding the path of the file to the `septic.ignored.paths` setting. . `*` can be used as a wildcard i.e. you can write `/templates/*` to ignore all files in the folder `templates`. The paths are calculated relative the workspace folder(s) root(s). This setting should be set on a workspace level in order avoid unintentionally ignoring files when switching between projects.
+-   Diagnostics (Errors/Warnings) for a certain line can be disabled by writing `// noqa` or `{# noqa #}` at the same line. It is recommended to use the jinja version when suppressing diagnostics when using scg in order to get proper diagnostics on the generated config. To disable certain errors/warnings write `{# noqa: Ennn, Wnnn #}` or `// noqa: Ennn, Ennn` (i.e. `noqa` + `:` + comma separated list of errors). The relevant error codes are displayed on the errors/warnings (e.g. `septic Enn`). To disable all diagnostics, use the `septic.diagnostics.enabled` setting. Diagnostics can be ignored for certain files by adding the path of the file to the `septic.ignored.paths` setting together with an array with the specific codes that should be ignored. Empty array is interpreted as ignore all codes.  `*` can be used as a wildcard i.e. you can write `/templates/*` to ignore all files in the folder `templates`. Examples: `"/templates_sim/Sopc*": []` will ignore all diagnostics from files that matches the pattern, `"/templates_main/SmpcAppl.cnfg": ["W501"]` will ignore all diagnostics with code W501. The paths are calculated relative the workspace folder(s) root(s). This setting should be set on a workspace level in order avoid unintentionally ignoring files when switching between projects.
```

Comment:
Suggest to amend this so that relative paths are according to standard *nix convention. I.e. `templates` and `./templates` are equivalent and refer to the directory `templates` in the current directory. Similarly `../templates` refers to a directory called `templates` at the same directory level as the current directory and `/templates/` is an absolute path. The latter should not be relevant for our use cases since we should always use relative paths.

Reply:
The updated version is treats `./templates/*` `/templates/*` and `templates/*` as the same